newIDE/app/src/PropertiesEditor/PropertiesMapToSchema.js

```diff
@@ -122,6 +122,34 @@ export default (
         },
         getLabel,
       };
+    } else if (valueType === 'color') {
```

Comment:
I wonder if we should use "hexColor" instead to be clear this property is stored as a hex string.